src/flint/FlintTypes.jl

```diff
@@ -48,7 +48,7 @@ mutable struct fmpz <: RingElem
     end
 
     function fmpz(x::Float64)
-        !isinteger(x) && throw(InexactError())
+        !isinteger(x) && throw(InexactError(:convert, fmpz, x))
```

Comment:
You could also write `InexactError(:fmpz, fmpz, x))`, and then the message will be something like `InexactError: fmpz(1.1)`.

Reply:
unfortunately that does not seem to be the case, at least not in Julia 1.

Reply:
Ah right sorry, it starts to have this behavior in Julia 1.1 ... I find the API of this `InexactError` function weird in any case.